### DISCUSSION TITLE
chdman

### DISCUSSION THREAD

**Discussion by tjanas** [retronas/discussions#15] [General]
Would it be possible or feasible to install chdman to be an optional tool? So far, I haven’t found a way to install it on a Linux machine without installing the mame-tools package, which is too large (over 800MB) for a retronas machine.

**Comment by sairuk** [comment]
we'd like too but building things like mames tooling on an rpi(4) is slow. we have this as something we want to do on the ideas page already along with other media management, for now we'd be dependent on external repos providing an artifact we can pull

for rpi pulling from dans builds might be an option but he'd need to provide a consistent name for latest

https://stickfreaks.com/mame/
https://github.com/danmons/retronas/wiki/Ideas#tools
